app/controllers/provider/sessions_controller.rb

```diff
@@ -33,6 +33,11 @@ def create
     end
   end
 
+  def bounce
+    auth = site_account.self_authentication_providers.find_by!(system_name: params[:system_name])
```

Comment:
`params.require(:system_name)` ?

Reply:
ok 😄 although it must exist anyway because it is part of the routing and otherwise it wouldn't find the route 😄